core/src/database/migrations.cpp

```diff
@@ -674,5 +674,13 @@ namespace ledger {
 
         template <> void rollback<14>(soci::session& sql) {
         }
+
+        template <> void migrate<15>(soci::session& sql) {
+            sql << "ALTER TABLE tezos_accounts RENAME COLUMN address TO public_key";
```

Comment:
Wait. If you don’t change the `public_key -> address` line 589… that PR is not even needed, right?

Reply:
Initially it was address and it seems that Live is using already a version with XTZ in prod ... so ...